python.cmake

```diff
@@ -351,6 +349,11 @@ macro(FINDPYTHON)
     endif()
   endif()
 
+  # Avoid Windows path
```

Comment:
This comment confused me a bit. Maybe something a bit more explicit like:
`# Avoid paths in Windows format`